### ISSUE TITLE
Proton Compatibility hit or miss

### ISSUE THREAD

**Issue by SSchott** [ChimeraOS/chimeraos#519]
While trying to setup the compatibility layer in the new UI, things seem to get very unstable. The option selected for all games is not stored, and for specific games gets deactivated randomly. For Elden Ring, I am not sure how or what exactly I do, but is a mixture of restarting and enabling/disabling the global and specific option. Any way of fixing this?

**Comment by digitalaiz** [comment]
i believe it's better to setup the compatibility layer outside of gamepad UI mode ( regular steam app in desktop mode ) .for me the best way is setting experimental as default and then adding custom GE versions per game . 
Not sure if that was your issue but i hope it helped

**Comment by SSchott** [comment]
I'll check if doing it that way, makes the changes being persistent. Thanks for the suggestion.

**Comment by SSchott** [comment]
Today I had time to test around with Desktop mode, and changes to Steam Play options there are also non-persistent, i.e., whatever I choose, it restarts Steam, to show the Proton option unchecked afterwards. The per game compatibility option seems to work though.

**Comment by SSchott** [comment]
No, the per game compatibility is also not persistent. @Samsagax is there anything you guys can check about this?
